src/pykeen/utils.py

```diff
@@ -1255,6 +1256,16 @@ def boxe_kg_arity_position_score(
     return negative_norm(element_wise_distance, p=p, power_norm=power_norm)
 
 
+def tensor_to_set(tensor: torch.LongTensor) -> Set[Tuple[int, ...]]:
+    """Convert a tensor of triples to a set of int-tuples."""
+    return set(map(tuple, tensor.tolist()))
```

Comment:
we _must_ already have this function somewhere.

Reply:
I looked for `.tolist(` and found the following:
* https://github.com/pykeen/pykeen/blob/6769898b496c5ba0c2b69161f633b7b4536a60e5/src/pykeen/models/predict.py#L603
* https://github.com/pykeen/pykeen/blob/6769898b496c5ba0c2b69161f633b7b4536a60e5/src/pykeen/sampling/filtering.py#L203
* https://github.com/pykeen/pykeen/blob/6769898b496c5ba0c2b69161f633b7b4536a60e5/src/pykeen/triples/triples_factory.py#L1186-L1187

Reply:
552da69

Reply:
smt = set map tuple lol. thanks for checking